example_site/src/components/AccountAuthentication.js

```diff
@@ -74,38 +76,39 @@ function LoginAccountForm() {
   };
 
   return (
-    <div className="panel">
+    <div>
       <form data-test="loginAccount-form" onSubmit={handleSubmit(onSubmit)}>
-        <h2>waymark.loginAccount()</h2>
+        <h2>Log in to existing account</h2>
         <p>
-          Either account ID or external ID may be used, but only one at a time.
+          Find an account with your external ID or Waymark account ID
         </p>
-        <label className="form-label" htmlFor="loginAccountAccountID">
-          Account ID
+
+        <label className="form-label" htmlFor="loginAccountExternalID">
+          External ID
         </label>
         <input
           type="text"
           className="form-input"
-          id="loginAccountAccountID"
-          name="accountID"
-          placeholder="eg. 1234-ABCD-1234-ABCD"
+          id="loginAccountExternalID"
+          name="externalID"
+          placeholder="<partner account ID>"
           ref={register}
         />
 
-        <label className="form-label" htmlFor="loginAccountExternalID">
-          External ID
+        <label className="form-label" htmlFor="loginAccountAccountID">
+          Account ID
         </label>
         <input
           type="text"
           className="form-input"
-          id="loginAccountExternalID"
-          name="externalID"
-          placeholder="<partner account ID>"
+          id="loginAccountAccountID"
+          name="accountID"
+          placeholder="eg. 1234-ABCD-1234-ABCD"
           ref={register}
         />
 
-        <button className="submit-button" data-test="loginAccount-button">
-          Login
+        <button className="submit-button form-input" data-test="loginAccount-button">
```

Comment:
Are you just applying the`form-input` class to get the correct width?

Since this is a button (not a form input) I would define a separate class in `App.css` called `form-button` that you can apply here instead.

```
form-input,
form-button {
    width: 20em;
}
```